### ISSUE TITLE
Feature Request: Display preview of themes inline in the theme-installation window

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [atom/atom#15895]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

At the moment, I have to resort to a browser tab to hunt down an Atom theme that I like - the "Install Themes" window is fairly useless until I *already know* which theme I want.

Thus, I'd like to suggest that that window display a row of four or five tiny, rendered, code-editors below each theme in the list; each of which displays an example of syntax-highlighting with a different language / snippet.

(If that's too architecturally complex, then perhaps simply inlining the first image found on the project's GitHub README is a better solution?)

tl;dr I think the theme-browser can be made more useful! `(=`

**Comment by rsese** [comment]
Thanks for the suggestion!

Enhancement requests for the settings view should go in https://github.com/atom/settings-view -
 I searched around there and found that this is a duplicate of https://github.com/atom/settings-view/issues/106 so you can subscribe there if you'd like.

---

Because we treat our issues list as [the Atom team's backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), we close duplicates to focus our work and not have to touch the same chunk of code for the same reason multiple times. This is also why we may mark something as duplicate that isn't an exact duplicate but is closely related.

For information on how to use GitHub's search feature to find out if something is a duplicate before filing, see [the How Can I Contribute? section of the Atom CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-can-i-contribute).
